frontend/src/metabase/components/InputBlurChange.styled.tsx

```diff
@@ -1,16 +1,21 @@
 import styled from "@emotion/styled";
+import {
+  inputFocusOutline,
+  inputPadding,
+  inputTypography,
+  numericInputReset,
+} from "metabase/core/style/input";
 import { color } from "metabase/lib/colors";
 
 export const Input = styled.input`
-  border: 1px solid ${color("border")};
+  ${inputPadding};
+  ${inputTypography};
+  border: 1px solid ${() => color("border")};
+  border-radius: 0.5rem;
+  color: ${() => color("text-dark")};
+  transition: border 0.3s;
```

Comment:
do these css properties need to be between the 4 imported styles? If not we lay all the imported styles next to each other just like you do in `frontend/src/metabase/visualizations/components/settings/ChartSettingInputNumeric.styled.tsx` for consistency?